lib/hanami/cli/generators/gem/app/assets.mjs

```diff
@@ -0,0 +1,14 @@
+import * as assets from "hanami-assets";
+
+await assets.run();
+
+// To provide additional esbuild (https://esbuild.github.io) options, use the following:
```

Comment:
@timriley Good for now. We'll need to replace this code snipped with a link to a future Hanami Guide article. 😉

Reply:
Good call. Have made [a draft issue](https://github.com/orgs/hanami/projects/2/views/1?pane=issue&itemId=41340063) in our project to remind us.